### ISSUE TITLE
Fay Error: weakly-referenced object no longer exists

### ISSUE THREAD

**Issue by liujin163** [waityousea/xuniren#26]
运行 fay_connect.py 在生成视频步骤的时候报错：Fay Error: weakly-referenced object no longer exists

message: D:/project/github/Fay/samples/sample-1692692582857.mp3 <class 'str'>
[WARN] audio sample rate is 24000, resampling into 16000.
[INFO] loaded audio stream data/audio/aud_1692692584.wav: (178176,)
[START]
 mi ha bo ŝi ĉafiĝi al en pa liu pi li reci i jemu sin ka faregon kun ŝi non ĉuŝa odeg aŭnu ŝi lejun hutikon ŝitan de dafu re ĉiĝi
[END]
[INFO] save all feats for training purpose...
[INFO] saved logits to data/audio/aud_1692692584_eo.npy
[INFO] load 3508 frames.
[INFO] load data/audio/aud_1692692584_eo.npy aud_features: torch.Size([279, 44, 16])
Loading data: 100%|██████████████████████████████████████████████████████████████| 3508/3508 [00:00<00:00, 92562.93it/s]
[INFO] eye_area: 0.25 - 0.25
==> Start Test, save results to data/video\results
***** 1
  0% 0/279 [00:00<?, ?it/s]***** 2
***** 3
<torch.utils.data.dataloader.DataLoader object at 0x00000257575F1630>
Fay Error: weakly-referenced object no longer exists
